### PR TITLE
common prometheus-pre7: support_groups and service added on alerts

### PR DESCRIPTION
**Pull request by richardtief**
Prerequisite for onboarding the global prometheus to the new operating model. The same applies to the Maia prometheus, with special routing now possible.

### CODE REVIEW

File: common/prometheus-server-pre7/templates/alerts/_prometheus.alerts.tpl [hunk 1]
```diff
@@ -24,131 +20,122 @@ groups:
...
     labels:
       context: availability
-      service: metrics
+      service: {{ default "metrics" .Values.alerts.service }}
       support_group: observability
       severity: critical
-      tier: {{ include "alerts.tier" . }}
       playbook: 'docs/support/playbook/prometheus/failed_config_reload.html'
-      meta: 'Prometheus {{`{{ $labels.prometheus }}`}} failed to load it`s configuration.'
+      meta: 'Prometheus `{{`{{ $labels.prometheus }}`}}` failed to load it`s configuration.'
```
Comment (viennaa) [inline common/prometheus-server-pre7/templates/alerts/_prometheus.alerts.tpl:27]: Why do we need to double tick this?

**Review by viennaa — COMMENTED** [review]
Thanks for preparing the move. Two little questions from my side.

File: common/prometheus-server-pre7/templates/alerts/prometheus-alerts.yaml [hunk 2]
```diff
@@ -5,7 +5,7 @@ kind: PrometheusRule
 metadata:
   name: {{ include "prometheus.fullName" . }}-prometheus.alerts
   labels:
-    prometheus: {{ include "prometheus.name" . }}
+    prometheus: {{ default (include "prometheus.name" .) .Values.alerts.prometheus }}
```
[the remark this answers is not in the document]
Comment (richardtief) [reply common/prometheus-server-pre7/templates/alerts/prometheus-alerts.yaml:8]: For Maia, we can set `prometheus-openstack` to pick up the metrics from Maia Prometheus and Thanos. This way we can remove the Alertmanger from Maia Prometheus again.

**Review by viennaa — CHANGES_REQUESTED** [review]
Little typo fix

**Review by viennaa — APPROVED** [review]
I like it, let's go.

**Pull request merged**